app/assets/javascripts/blacklight/blacklight.js

```diff
@@ -327,19 +327,24 @@
     // to show to user in modal.  Right now called only for extreme
     // network errors.
     modal.onFailure = function(jqXHR, textStatus, errorThrown) {
-      console.error('Server error:', this.url, jqXHR.status, errorThrown);
-
-      var contents = '<div class="modal-header">' +
-                '<div class="modal-title">There was a problem with your request.</div>' +
-                '<button type="button" class="blacklight-modal-close btn-close close" data-dismiss="modal" aria-label="Close">' +
-                '  <span aria-hidden="true">&times;</span>' +
-                '</button></div>' +
-                ' <div class="modal-body"><p>Expected a successful response from the server, but got an error</p>' +
-                '<pre>' +
-                this.type + ' ' + this.url + "\n" + jqXHR.status + ': ' + errorThrown +
-                '</pre></div>';
-      $(modal.modalSelector).find('.modal-content').html(contents);
-      modal.show();
+        console.error('Server error:', this.url, jqXHR.status, errorThrown);
+
+        const contents = `<div class="modal-header">
+        <div class="modal-title">There was a problem with your request.</div>
+        <button type="button" class="blacklight-modal-close btn-close close" data-dismiss="modal" data-bs-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+        </div>
+        <div class="modal-body">
+          <p>Expected a successful response from the server, but got an error</p>
+          <pre>${this.type} ${this.url}\n${jqXHR.status}: ${errorThrown}</pre>
+        </div>`;
+        // const domparser = new DOMParser();
+        // const dom = domparser.parseFromString(contents, "text/html")
+        // const nodes = dom.querySelectorAll('body > *')
```

Comment:
@jcoyne is this left here for documentation purposes?

Reply:
Oops, this is cruft. I'll pull it.

Reply:
The files in app/assets/javascript are just compiled versions (using `npm run js-compile-bundle`) of what is in `app/javascript`.  In this instance I didn't recompile after changing src.

Reply:
I recompiled and pushed the results up. That had the effect of removing these old comments.